packages/split-button/src/SplitButton/SplitButton.spec.tsx

```diff
@@ -205,22 +209,32 @@ describe('packages/split-button', () => {
       document.body.appendChild(portalContainer);
       const portalRef = createRef<HTMLElement>();
       renderSplitButton({
-        open,
+        open: true,
```

Comment:
This was passing a global `open` function as prop, which was only revealed when restricting type of the argument passed to `renderSplitButton`.